Lib/fontbakery/profiles/googlefonts.py

```diff
@@ -4129,23 +4129,28 @@ def typo_metrics_enabled(ttFont):
 def com_google_fonts_check_varfont_instance_coordinates(ttFont):
   """Check variable font instances have correct coordinate values"""
   from fontbakery.parse import instance_parse
+  from fontbakery.constants import SHOW_GF_DOCS_MSG
   failed = False
   for instance in ttFont['fvar'].instances:
-    name = ttFont['name'].getName(instance.subfamilyNameID,
-                                  PlatformID.WINDOWS,
-                                  WindowsEncodingID.UNICODE_BMP,
-                                  WindowsLanguageID.ENGLISH_USA).toUnicode()
+    name = ttFont['name'].getName(
+      instance.subfamilyNameID,
+      PlatformID.WINDOWS,
+      WindowsEncodingID.UNICODE_BMP,
+      WindowsLanguageID.ENGLISH_USA
+    ).toUnicode()
     expected_instance = instance_parse(name)
     for axis in instance.coordinates:
-      if instance.coordinates[axis] != expected_instance.coordinates[axis]:
+      if axis in expected_instance.coordinates and \
+        instance.coordinates[axis] != expected_instance.coordinates[axis]:
+        yield FAIL, (
+          f'Instance "{name}": "{axis}" value "{instance.coordinates[axis]}" '
+          f'is incorrect, it should be '
+          f'"{expected_instance.coordinates[axis]}"')
         failed = True
-        yield FAIL,\
-              Message("bad-coordinate",
```

Comment:
I noticed that you removed the message-IDs in this patch (in several places, here's one of them). This makes our code-testing less strict. I'd prefer to keep the message IDs so that we explicitly test that certain bad fonts cause very specific FAILs or WARNs

Reply:
Other than this detail in code-testing approach, I am very happy to see progress in this. Thanks, @m4rc1e!

Reply:
Ok no problem.

Reply:
I've restored the Messages. I'm meant to be working on the name checks next quarter so I should added further improvements soonish.